packages/http-client-python/test/unbranded/mock_api_tests/cadl-ranch-config.yaml

```diff
@@ -0,0 +1,27 @@
+unsupportedScenarios: Azure_ClientGenerator_Core_Access_InternalOperation
```

Comment:
is this actually the right syntax?

Reply:
i'm not sure actually, let me remove for now bc we actaully do support this